src/multigifts.cpp

```diff
@@ -726,8 +726,7 @@ void recvMultiPlayerFeature(NETQUEUE queue)
 		if (asFeatureStats[i].ref == ref)
 		{
 			// Create a feature of the specified type at the given location
-			FEATURE *result = buildFeature(&asFeatureStats[i], x, y, false);
-			result->id = id;
+			buildFeature(&asFeatureStats[i], x, y, false);
```

Comment:
Seems like `buildFeature` might need the same wrapper as you applied to `buildStructure`? (And in this case, would pass the id in?)

Reply:
you are correct, I haven't noticed `id` above